src/io/windows.zig

```diff
@@ -1220,108 +1277,3 @@ fn getsockoptError(socket: os.socket_t) IO.ConnectError!void {
         else => |e| os.windows.unexpectedWSAError(e),
     };
 }
-
-// Vendor std.os.windows.OpenFile so we can set file attributes. Add it as a parameter after
-// `options`, so we don't have to vendor that struct too.
-pub fn windows_open_file(
```

Comment:
Ah, it seems like https://github.com/tigerbeetle/tigerbeetle/pull/1925 got lost in the rebase; could you bring it back in?

Reply:
Sure. IIRC, `FILE_FLAG_OVERLAPPED` being passed through attributes was causing an unexpected "invalid value" error. Likely conflicting flags so will bump a fix.

Reply:
I'm not sure I follow why, but [Chrome](https://chromium.googlesource.com/chromium/chromium/+/refs/heads/trunk/chrome_elf/create_file/chrome_create_file.cc#180) seem to translate `FILE_FLAG_OVERLAPPED` as `FILE_SYNCHRONOUS_IO_NONALERT`

Reply:
(the `FILE_FLAG_*` flags are only for the kernel32 CreateFile, ntcreatefile deals with `FILE_*` :/)

Reply:
Seems like some of attributes are being passed to the wrong NtCreateFile param (`file_flags` vs `attributes`).